sources/data-node/grpc-rest-bindings.yml

```diff
@@ -131,10 +126,6 @@ http:
       get: "/liquidity-provisions/party/{party}/market/{market}"
 
     # Miscellaneous
-    - selector: datanode.api.v1.TradingDataService.Statistics
-      get: "/statistics"
-    - selector: datanode.api.v1.TradingDataService.LastBlockHeight
-      get: "/blockchain/height"
```

Comment:
Does this endpoint still exist after being removed from this file? I've used it recently on data-node, but I see that it doesn't actually exist on the `TradingDataService`

Reply:
I think so, it gets proxied to the core service